esp-hal/src/spi/master.rs

```diff
@@ -666,7 +696,9 @@ where
 
 impl<'d> Spi<'d, Blocking> {
     /// Constructs an SPI instance in 8bit dataframe mode.
-    // FIXME: when https://github.com/esp-rs/esp-hal/issues/2839 is resolved, add an appropriate `# Error` entry.
+    ///
+    /// # Errors
+    /// See [`Spi::apply_config`].
```

Comment:
```suggestion
    /// # Errors
    ///
    /// See [`Spi::apply_config`].
```

Just to be consistent with the other comments, Ill leave an empty line after the header